src/organisms/CollapsibleTable/CollapsibleTable.tsx

```diff
@@ -0,0 +1,228 @@
+import throttle from 'lodash/throttle';
+import React, {
+  Component,
+  DetailedHTMLProps,
+  HTMLAttributes,
+  ReactNode,
+} from 'react';
+import { StyledComponentClass } from 'styled-components';
+
+import { Icon } from 'src/atoms';
+import {
+  StackedCard,
+  StackedCardData,
+  Table,
+  TableConfig,
+  TableData,
+} from 'src/molecules';
+import styled from 'src/styled-components';
+import Theme, { scale } from 'src/Theme';
+import Typography from 'src/Typography';
+
+export enum CollapsibleTableModes {
+  Mobile,
+  Desktop,
+}
+
+export interface CollapsibleTableConfig extends TableConfig {
+  primaryColumn: string;
+  iconColumns?: string[];
+}
+
+export interface CollapsibleTableData extends TableData {
+  config: CollapsibleTableConfig;
+}
+
+interface CollapsedGroups {
+  [title: string]: boolean;
+}
+
+type StyledHTMLElement = StyledComponentClass<
+  DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>,
+  Theme
+>;
+
+interface Flippable {
+  isFlipped?: boolean;
+}
+
+interface Props extends CollapsibleTableData {
+  breakpoint: number;
+}
+
+interface State {
+  mode: CollapsibleTableModes;
```

Comment:
Since there are only two modes, would it be easier to have an `isLabeled` property?

Reply:
Maybe, but I don't think this harms readability/usability at all, and allows for more modes depending on other conditions.

Reply:
I'm thinking we'd only need two, because even if the media queries change it would only toggle between table and stacked card.